.github/workflows/release-docs.yml

```diff
@@ -0,0 +1,59 @@
+---
+name: Build release docs
+
+on:
+    # Allows you to run this workflow manually from the Actions tab
+    # It should be run right after tagging the release commit on main
+    workflow_dispatch:
+
+jobs:
+    release-docs:
+        # if: github.repository == 'nilearn/nilearn' && contains('["ymzayek","Remi-Gau"]', github.actor)
+        if: github.repository == 'ymzayek/nilearn' && github.actor == 'ymzayek'
+        runs-on: ubuntu-latest
+        timeout-minutes: 360
+        env:
+            BROWSER: /usr/bin/firefox
+            DISPLAY: :99.0
+            NILEARN_DATA: /home/runner/work/nilearn/nilearn/nilearn_data
+        defaults:
+            run:
+                shell: bash -el {0}
+
+        steps:
+        -   name: Checkout nilearn
+            uses: actions/checkout@v4
+
+        -   name: Install apt packages
+            run: |
+                sudo -E apt-get -yq update
+                sudo -E apt-get -yq --no-install-suggests --no-install-recommends install \
+                    dvipng texlive-latex-base texlive-latex-extra
+
+        -   name: Setup python
+            uses: actions/setup-python@v5
+            with:
+                python-version: '3.12'
+
+        -   name: Install packages
+            run: |
+                python -m pip install --user --upgrade pip setuptools
```

Comment:
Any reason setuptools was re-added during the tox implementation? @Remi-Gau

Reply:
For reference I copied this line from build-docs.yml which was changed during the tox PR but we had removed setuptools when we switched to hatchling backend for builds

Reply:
Going from memory, but pretty sure it got needed when running the doc build on python 3.12.

May be worth rechecking and if that's indeed the case documenting it somewhere.

Reply:
Ok sorry I missed that. Yes it seems to be the case. Can add to the readme